types/pollyjs__core/index.d.ts

```diff
@@ -1,181 +1,211 @@
-// Type definitions for @pollyjs/core 2.6
+// Type definitions for @pollyjs/core 2.6.2
 // Project: https://github.com/netflix/pollyjs/tree/master/packages/@pollyjs/core
 // Definitions by: feinoujc <https://github.com/feinoujc>
 //                 Borui Gu <https://github.com/BoruiGu>
+//                 Offir Golan <https://github.com/offirgolan>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.4
 
-import { MODES } from '@pollyjs/utils';
 import Adapter from '@pollyjs/adapter';
```

Comment:
Should these imports be added as dependencies in package.json?

Reply:
I think just doing this makes dt add these as dependencies so there is no need to do that

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/27f6e1e2e7e56f307169cb25e80cebac6a460fc9/types/pollyjs__core/tsconfig.json#L22-L30